### PR TITLE
Percept my_stakeholder_id(<ID>)

### PR DESCRIPTION
**Pull request by klikooo**
With this we can percept `my_stakeholder_id(<ID>)`, this fixes issue #11 

Travis buid log: https://travis-ci.org/Denpeer/tygron/builds/133357468

User story: 
As a developer, I want the virtual human to be able to see who it is, so that it can query information about itself, its indicators and its belongings.

### CODE REVIEW

File: environment/src/main/java/tygronenv/EisEnv.java [hunk 1]
```diff
@@ -66,6 +66,7 @@ public EisEnv() {
 
 	@Override
 	protected LinkedList<Percept> getAllPerceptsFromEntity(String e) throws PerceiveException, NoEnvironmentException {
+
```
Comment (dionytadema) [inline environment/src/main/java/tygronenv/EisEnv.java]: why did this end up in the changes? could you please undo this change

File: environment/src/main/java/tygronenv/TygronEntity.java [hunk 2]
```diff
@@ -96,7 +95,8 @@ public void notifyReady(String entity) throws EntityException {
 	 *             stakeholder name does not exist.
 	 * 
 	 */
-	private void connectStakeholder() throws EntityException {
+
```
Comment (dionytadema) [inline environment/src/main/java/tygronenv/TygronEntity.java]: please don't put whitelines between javadoc and the method it documents

**Comment by klikooo** [comment]
Removed them @dionytadema ..

**Comment by jannelie** [comment]
We approve -CrashTestDummies

**Comment by dionytadema** [comment]
We approve -DaNSHaL, however a test could still be added

**Comment by klikooo** [comment]
What kind of tests besides the tests that are already in this pr? @dionytadema

**Comment by levilime** [comment]
If I've deduced correctly this PR killed request percept behaviour. Which will be #62 to fix. This was not ready to be merged, too bad my team gave approval :smile:

File: environment/src/main/java/tygronenv/TygronEntity.java [hunk 3]
```diff
@@ -79,7 +79,6 @@ public TygronEntity(EisEnv env, String intendedStakeholder, Integer slotID) {
 	 * @throws EntityException
 	 */
 	public void notifyReady(String entity) throws EntityException {
-		connectStakeholder();
```
Comment (levilime) [inline environment/src/main/java/tygronenv/TygronEntity.java:82]: This ommition broke the requests percept
